### ISSUE TITLE
Unable to upload small files to glacier

### ISSUE THREAD

**Issue by lyenliang** [tbumi/glacier-upload#29]
I got the following error when uploading a file whose size is less than 4MB

```
botocore.errorfactory.InvalidParameterValueException: An error occurred (InvalidParameterValueException) when calling the UploadArchive operation: Invalid Content-Length: 0
```
This error happens at line 70 of `upload.py`, which is `body=file_to_upload,`.

The problem seems to be related to `file_size = file_to_upload.seek(0, 2)` line in `upload.py`. 

Because after I change it to `file_size = 1`, the problem no longer occurs.

**Comment by awaismehmood88** [comment]
replace 
     

> file_size = file_to_upload.seek(0, 2) 

with
> import os
>## file_size = os.stat(file_name[0]).st_size

**Comment by tbumi** [comment]
Fixed in [version 2.0](https://pypi.org/project/glacier-upload/2.0/).
